src/app/pages/registration/registration-form/registration-form.component.html

```diff
@@ -5,99 +5,74 @@ <h2>Registration Form</h2>
-               maxlength="255" email>
-        <div class="alert alert-danger" role="alert" *ngIf="email.errors?.required && (email.touched || submitted)">
+        </mat-error>
+      </mat-form-field>
+      <label>sex*: </label>
+      <mat-radio-group formControlName="sex">
+        <mat-radio-button type="radio" name="sex" value="F" required> female</mat-radio-button>
+        <mat-radio-button type="radio" name="sex" value="M" required> male</mat-radio-button>
+        <mat-error *ngIf="sex.invalid && (sex.dirty || sex.touched || submitted)">
           This field is required
-        </div>
-        <div class="alert alert-danger" role="alert" *ngIf="email.errors?.email && (email.touched || submitted)">
-          This is not valid email
-        </div>
-      </div>
-      <div class="form-group required">
-        <label for="city">city</label>
-        <input type="text" id="city" class="form-control" [(ngModel)]="model.city" #city="ngModel" name="city" required
-               maxlength="255">
-        <div class="alert alert-danger" role="alert" *ngIf="city.errors?.required && (city.touched || submitted)">
+        </mat-error>
+      </mat-radio-group>
+      <mat-form-field>
+        <mat-select name="size" id="size" formControlName="size" required placeholder="t-shirt size">
+          <mat-option></mat-option>
+          <mat-option value="S">S</mat-option>
+          <mat-option value="M">M</mat-option>
+          <mat-option value="L">L</mat-option>
+          <mat-option value="XL">XL</mat-option>
+          <mat-option value="XXL">XXL</mat-option>
+        </mat-select>
+        <mat-error *ngIf="size.invalid && (size.dirty || size.touched || submitted)">
           This field is required
-        </div>
-      </div>
-      <div class="form-group required">
-        <label>sex</label>
-        <label class="checkbox">
-          <input type="radio" name="sex" value="F" required [(ngModel)]="model.gender" #sex="ngModel"> female
-        </label>
-        <label class="checkbox">
-          <input type="radio" name="sex" value="M" required [(ngModel)]="model.gender" #sex="ngModel"> male
-        </label>
-        <div class="alert alert-danger" role="alert" *ngIf="sex.errors?.required && (sex.touched || submitted)">
+        </mat-error>
+      </mat-form-field>
+      <mat-form-field>
+        <mat-select name="experience" id="experience" formControlName="experience" required placeholder="your experience">
+          <mat-option></mat-option>
+          <mat-option value="I am just learning">I am just learning</mat-option>
+          <mat-option value="Less than a year">Less than a year</mat-option>
+          <mat-option value="1-2 years">1-2 years</mat-option>
+          <mat-option value="2-4 years">2-4 years</mat-option>
+          <mat-option value="4-8 years">4-8 years</mat-option>
+          <mat-option value="Over 8 years">Over 8 years</mat-option>
```

Comment:
I am wondering if we can ask this data now (GDPR). We dont need this data to do our business, so for me it is questionable.